client/src/use/useAnnotationLayer.js

```diff
@@ -17,12 +17,12 @@ export default function useAnnotationLayer({
       strokeColor: (a, b, data) => {
         if (_editingTrackId !== null) {
           if (_editingTrackId !== data.record.detection.track) {
-            return stateStyling.disabled.color; // color for other objects when editing a detection
+            if (stateStyling.disabled.color) {
+              return stateStyling.disabled.color;
+            }
```

Comment:
`stateStyling.disable.color` is now removed so there isn't one in there.  It is defensive in that I wanted to be able to modify colors or styling across the geoJS layers independent of going into each one and changing things.  Like with the new UI/UX stuff we talked about not having a selected color and it being a different in opacity and thickness.  You simply remove the color from the stateStyling and defaults back to use the `typeColorMap` for  a selected item and allows you change the thickness and opacity instead.  

The falling through to line 31 is the proper event. Although I'm going to update it to match the comboValue in TrackItem.vue so it is the undefined color;  That is the fallback color of the root `typeColorMap` which occurs when you create a new track and don't have a type assigned  You are correct that the outside if statement should be combined with the internal one although the `null` check needs to remain explicit.  

You will notice in useTextLayer.js that it doesn't check the `confidencePairs.length` and that's because it is done in the data provider for the text layer.  Text doesn't render unless there is some confidencePair data associated with it.